connect-base/2.4/Dockerfile

```diff
@@ -42,6 +44,14 @@ RUN mkdir "$KAFKA_CONNECT_PLUGINS_DIR" "$EXTERNAL_LIBS_DIR"
 #
 RUN docker-maven-download apicurio "$APICURIO_VERSION" ead18a95038adca54e91b7f253717eb7
 RUN docker-maven-download central org/jolokia jolokia-jvm "$JOLOKIA_VERSION" d489d62d1143e6a2e85a869a4b824a67
+RUN docker-maven-download central io/opentelemetry opentelemetry-api "$OPENTELEMETRY_VERSION" e198a9568ce31a82faaa26f328388e89
+RUN docker-maven-download central io/opentelemetry opentelemetry-context "$OPENTELEMETRY_VERSION" 783594a506dbf035e686776d5bcb4bfc
```

Comment:
as you use later on `otel` directory (when calling `set_connector_additonal_resource_availability`), the bits needs to be downloaded into `otel` directory as well, i.e. you need too replace `central` with `otel`